extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeChannelExtractorTest.java

```diff
@@ -916,8 +917,14 @@ public void testVerified() throws Exception {
         @Test
         @Override
         public void testTabs() throws Exception {
-            // Gaming topic channels tabs are not yet supported, so an empty list should be returned
-            assertTrue(extractor.getTabs().isEmpty());
+            // Gaming topic channels tabs are not yet supported
+            // However, a Shorts tab like on other channel types is returned, so it is supported
+            // Check that it is returned
+            final List<ListLinkHandler> channelTabs = extractor.getTabs();
+            assertEquals(1, channelTabs.size());
+            final List<String> contentFilters = channelTabs.get(0).getContentFilters();
+            assertEquals(1, contentFilters.size());
+            assertEquals(ChannelTabs.SHORTS, contentFilters.get(0));
```

Comment:
Maybe change `assertTabsContain` so that:
- it asserts that the number of tabs is the same as the number of parameters passed
- it checks that every tab has exactly one content filter

And then do this:

```suggestion
            assertTabsContain(extractor.getTabs(), ChannelTabs.SHORTS);
```